src/ibek/params.py

```diff
@@ -17,7 +17,7 @@
 ]
 
 
-class DefinesTypes(Enum):
+class DefineTypes(Enum):
```

Comment:
I think this still has one too many `s`s

Reply:
agreed